### PR TITLE
Add weekly update 2016-04-17

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

File: locale/en/blog/weekly-updates/weekly-update.2016-04-17.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+status: publish
+category: weekly
+slug: weekly-update-2016-04-17
+layout: blog-post.hbs
+---
+
+### Node.js News
+Node v4.4.3 (LTS) is released.
+
+### Node v4.4.3 (LTS) Releases
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-04-17.md]: `Releases` -> `Release`

File: locale/en/blog/weekly-updates/weekly-update.2016-04-17.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+We have one releases: [Node v4.4.3 (LTS)](https://nodejs.org/en/blog/release/v4.4.3/). Complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-04-17.md]: `releases` -> `release`

File: locale/en/blog/weekly-updates/weekly-update.2016-04-17.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+### New Node.js Foundation Survey Reports New “Full Stack” In Demand Among Enterprise Developers
+
+The Node.js User Survey report features insights on emerging trends happening in this massive community that serves as a leading indicator on trends like microservices architectures, real-time web applications, Internet of Things (IoT). The report paints a detailed picture of the technologies that are being used, in particular, with Node.js in production and language preferences (current and future) for front end, back end and IoT developers.
+
+See https://nodejs.org/en/blog/announcements/nodejs-foundation-survey/ for more information.
+
+### Community Updates
+
+* [NodeUp podcast episode 100]: A Creative Coding Show with Mariko Kosaka, Kassandra Perch, Myles Borins, and Rich Trott.(http://nodeup.com/onehundred)
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-04-17.md]: Parentheses should come right after the brackets for Markdown to render correctly here...

File: locale/en/blog/weekly-updates/weekly-update.2016-04-17.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+* [A Quick Guide To Reading Node.js Core Source]: One person's approach to understanding the source code that makes up Node.js core. "Your mileage may vary. Warranty void if seal is broken." (https://medium.com/@Trott/a-quick-guide-to-reading-node-js-core-source-c968d83e4194#.mmontrmvg)
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-04-17.md]: ...and here.

**Comment by JungMinu** [comment]
Thanks, updated :)

**Pull request merged**